package.json

```diff
@@ -3,6 +3,9 @@
   "private": true,
   "version": "3.1.0",
   "type": "module",
+  "workspaces": [
```

Comment:
Admin should be a uniform project - we won't force monorepo on users just because of some e2e convenience.